bot.py

```diff
@@ -39,6 +36,444 @@ async def hello(ctx):
+    else:
+        not_roles = []
+
+    if only_nots_in_str is False:
+        role_names = (
+            discord_string.split(" and ")
+            if " and " in discord_string
+            else discord_string.split(" or ")
+        )
+        roles = []
+        for role_name in role_names:
+            role = discord.utils.get(ctx.guild.roles, name=role_name)
+            if role is not None:
+                roles.append(role)
+            else:
+                await ctx.send(f"Could not find `{role_name}` role. Check spelling.")
+                return
+        roles = [role for role in roles if role is not None]
+        if not roles:
+            await ctx.send("Could not find one or more roles. Check spelling.")
+            return
+
+    members = set()
+    if " and " in discord_string:
+        role_names = " and ".join(role.name for role in roles)
+        for role in roles:
+            if not members:  # start if members is empty
+                members.update(role.members)
+            else:
+                members &= set(role.members)
+    elif " or " in discord_string:
+        role_names = " or ".join(role.name for role in roles)
+        for role in roles:
+            members.update(role.members)
+    elif only_nots_in_str is False:  # only one positive role
+        role = roles[0]
+        members.update(role.members)
+        role_names = role.name
+    else:
+        role_names = role_names[0]
+        members = set(ctx.guild.members)
+
+    for not_role in not_roles:
+        members -= set(not_role.members)
+
+    members = list(members)
+    member_names = ", ".join(member.mention for member in members)
+    not_role_names = ", ".join(not_role.name for not_role in not_roles)
+    if not_roles:
+        if only_nots_in_str is True:
+            await ctx.send(
+                f"The members without roles `{not_role_names}` are: {member_names}."
+            )
+        else:
+            await ctx.send(
+                f"The members with the roles `{role_names}`"
+                f" but not the roles `{not_role_names}` are: {member_names}."
+            )
+    else:
+        await ctx.send(
+            f"The members with the roles `{role_names}` are: {member_names}."
+        )
+"""
+
+"""
+def converting_new(input_new):
+    converted_input = f"{input_new} + new segment"
+    return converted_input
+
+
+def adding_owner(ctx, converted_input):
+    owner = ctx.guild.owner.mention
+    final_message = f"{converted_input} + {owner}"
+    return final_message
+
+
+@bot.command()
+async def show_final_message_in_embed(ctx):
+    input_new = "12345"
+    converted_input = converting_new(input_new)
+    final_message = adding_owner(ctx, converted_input)
+
+    embed_new = discord.Embed(
+        title="Info",
+        description="Info",
+        color=discord.Colour.blue(),
+    )
+    embed_new.add_field(name="Final message", value=f"{final_message}", inline=True)
+    await ctx.send(embed=embed_new)
+"""
+
```

Comment:
Ten zakomentowany kod ma tu być? Czy też zapomniałeś go usunąć?

Reply:
To, a przynajmniej funkcje list_members i count_members, to początkowe funkcje, które działają w bocie i zastanawiałem się czy je zostawić zakomentowane, jakbym chciał szybko testować jakieś zmiany w funkcjach, ale widzę, że nie podobają się linterom ze względu na długość, więc w kolejnej wersji je usunę.

Reply:
Masz na myśli kolejną wersję bota, czy kolejną wersję tego brancha?